bioblend/galaxy/objects/wrappers.py

```diff
@@ -965,6 +1023,33 @@ def download(self, jeha_id, outf, chunk_size=bioblend.CHUNK_SIZE):
         return self.gi.gi.histories.download_history(
             self.id, jeha_id, outf, chunk_size=chunk_size)
 
+    def new_dataset_collection(self, collection_description):
```

Comment:
What about calling this `create_dataset_collection` instead? If you agree, please change also the corresponding test name.